src/components/Steps.tsx

```diff
@@ -81,7 +81,8 @@ const Steps = () => {
                         text={'create_account_text'}
                         btnText={'create_account'}
                         handleClick={handleCreateAccount}
-                        isDisabled={isWrongNetwork}
+                        // Disable for TVL feature
+                        isDisabled={true}
```

Comment:
This would break our testnet deployment

Reply:
So you want it so that on /deposit we should be on arbitrum one and when the user switches to the app part the app becomes arbitrum sepolia? In that case why would we disable the button at all? Isn't the point to completely disable testnet while the TVL feature is deployed?